react/components/Slider.js

```diff
@@ -402,25 +335,23 @@ class Slider extends Component {
   }
 
   onMouseMove = e => {
-    const { easing, loop, currentSlide, draggable, cursorOnMouseDown } = this.props
+    const { currentSlide, draggable, cursorOnMouseDown, easing } = this.props
 
     e.preventDefault()
     if (this.pointerDown && draggable) {
       // TODO prevent link clicks
 
       this.drag.endX = e.pageX
 
-      const computedCurrentSlide = loop ? currentSlide + this.perPage : currentSlide
-      const currentOffset = computedCurrentSlide * (this.selectorWidth / this.perPage)
+      const currentOffset = currentSlide * (this.selectorWidth / this.perPage)
       const dragOffset = (this.drag.endX - this.drag.startX)
-      const offset = currentOffset - dragOffset
+      const offset = (currentOffset - dragOffset) / this._sliderFrame.current.getBoundingClientRect().width * -100
```

Comment:
running `getBoundingClientRect` forces a re-layout, so running this on every `mouseMove` tick can be bad for performance. consider running this once and storing the value somewhere